src/main/scala/org/graphframes/lib/ConnectedComponents.scala

```diff
@@ -163,11 +163,11 @@ object ConnectedComponents extends Logging {
 
   import org.graphframes.GraphFrame._
 
-  private val COMPONENT = "component"
   private val ORIG_ID = "orig_id"
   private val MIN_NBR = "min_nbr"
   private val CNT = "cnt"
   private val CHECKPOINT_NAME_PREFIX = "connected-components"
+  val COMPONENT = "component"
```

Comment:
Why not to add an alternative merhod `run(graph, algorithm, broadcastThreshold, checkpointInterval, intermediateStorageLevel, componentColName)` instead and update the public API? Cause this one is a constant value that works fine for default, but it would be strange to see it as a part of the public API?